content/docs/06-integrations/00-aws-autoscaler.md

```diff
@@ -14,81 +14,174 @@ import InfoBox from 'shared/components/InfoBox';
+After updating your Kubernetes pack, add/update other layers per your requirements. Next, add AWS Cluster Autoscaler pack as an add-on layer. 
+<br />
+
+### Deploy a Cluster
+Once your cluster profile is complete, deploy it to a cluster. During the cluster configuration, ensure that your worker pool is made up of instance type **t3.2xlarge** (8 vCPUs, 32 GB RAM) or higher. As an example, the snapshot below shows one node of type **t3.2xlarge** in the worker pool of a successfully deployed cluster.  
+
+
+![Snapshot showing one node of type **t3.2xlarge** in the worker pool.](/integrations_aws-cluster-autoscaler_one-node.png)
+
+### Validate
+To validate the functioning of the AWS AutoScaler pack, navigate to the **Nodes** tab in the cluster details page in Palette. Notice the current nodes in the worker pool, and edit the configuration to scale down the instance type to **t3.medium** (2 vCPUs, 8 GB RAM). The snapshot below shows how to edit the instance type in the node pool configuration. 
+
+
+![Snapshot showing how to edit node pool configuration.](/integrations_aws-cluster-autoscaler_edit-node.png)
+
+
+Scaling down the instance type will result in insufficient resources for some of the existing pods. Consequently, Cluster Autoscaler will provision new nodes with smaller instance sizes, *collectively* providing enough capacity to accommodate the current workload. The snapshot below shows new nodes of size **t3.medium** spin up automatically.
+
+
+![Snapshot showing new nodes of size **t3.medium** spin up automatically, *collectively* providing enough capacity to accommodate the current workload. ](/integrations_aws-cluster-autoscaler_two-nodes.png)
+
+AWS Cluster Autoscaler dynamically scales cluster nodes to maximize cluster utilization by monitoring the workload. If nodes are overutilized, it will provision new nodes to accommodate unscheduled pods. On the other hand, if nodes are underutilized, it will migrate the pods from underutilized to other available nodes and shut down underutilized ones.   
+
+
+
+# Troubleshooting
+
+1. If you are facing the `LimitExceeded: Cannot exceed quota for PoliciesPerRole:10` error in the cluster deployment logs, it may be because the default IAM role Palette creates for the node group already has 10 policies attached to it, and you are trying to attach one more. By default, your AWS account will have a quota of 10 managed policies per IAM role. To fix the error, follow the instruction on this [AWS guide](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-quotas.html#reference_iam-quotas-entities) to request a quota increase. 
+
+
+2. If you encounter an `executable aws-iam-authenticator not found` error in your terminal while trying to access your EKS cluster from your local machine, you can resolve it by installing the 
+[aws-iam-authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html) plugin on your local machine. 

```

Comment:
Why are these two numbered? THat makes it seem like they are sequential steps. Let's remove the numbers.